script.js

```diff
@@ -0,0 +1,54 @@
+let chatVisible = false;
+
+function toggleChat() {
+    const chatContainer = document.querySelector('.chat-container');
+    chatVisible = !chatVisible;
+    chatContainer.style.display = chatVisible ? 'block' : 'none';
+}
```

Comment:
```suggestion
function toggleChat() {
    const chatContainer = document.querySelector('.chat-container');
    chatVisible = !chatVisible;
    chatContainer.style.display = chatVisible ? 'block' : 'none';
}
```
use arrow functions

Reply:
and also avoid using "querySelector" on "id's"